### DISCUSSION TITLE
Clarification on unstable features

### DISCUSSION THREAD

**Discussion by johnbellessa** [kube-rs/kube#1592] [Q&A] [answered]
I haven't seen this documented anywhere or answered in other questions, but apologies if I missed it (or if I'm just being dense)

I'm just wanting to confirm that `unstable` refers to a particular feature's API being unstable, not the functionality itself. Is that correct?

**Comment by clux** [comment, marked as answer]
The documentation is here: https://kube.rs/stability/#unstable-features

Generally the desire is what you say above, but it's hard to actually codify that. If you change the API you're likely also changing part of the functionality.

**Reply by johnbellessa** [reply]
Makes sense, thank you!
